scripts/validate-charts

```diff
@@ -51,7 +51,7 @@ check_system_registry() {
     chart_tmp=$3
 
     yaml_tmp=$(mktemp --suffix .yaml)
-    values="global.systemDefaultRegistry=my-registry,global.cattle.systemDefaultRegistry=my-registry,vCenter.clusterId=test-id"
+    values="global.systemDefaultRegistry=my-registry,global.cattle.systemDefaultRegistry=my-registry,vCenter.clusterId=test-id,global.clusterDNS=10.43.0.10"
```

Comment:
Can we validate this with dual-stack? Ref: https://github.com/rancher/rke2/issues/5156
```suggestion
    values="global.systemDefaultRegistry=my-registry,global.cattle.systemDefaultRegistry=my-registry,vCenter.clusterId=test-id,global.clusterDNS=10.43.0.10,2001:cafe:43::a"
```

Reply:
done